doc/api/dns.md

```diff
@@ -147,6 +139,12 @@ on some operating systems (e.g FreeBSD 10.1).
 <!-- YAML
 added: v0.11.14
 -->
+- `address` {string}
+- `port` {number}
+- `callback(err, hostname, service)` {Function}
+  - `err` {Error}
+  - `hostname` {string} e.g. `localhost`
```

Comment:
'example.com' would make a better example, localhost is a unusual host, it isn't even required to be resolved by a DNS server